deploy/scripts/src/ipv-core/test.ts

```diff
@@ -94,6 +95,8 @@ const profiles: ProfileList = {
 }
 
 const loadProfile = selectProfile(profiles)
+const iterationsStarted = new Counter('Iterations_Started')
+const iterationsCompleted = new Counter('Iterations_Completed')
```

Comment:
I think it would make sense to move these to a common utilities file and then import them. That way the metric name will be consistent across all tests that use these

Reply:
Done